packages/balances/src/BalanceModule.ts

```diff
@@ -153,6 +164,9 @@ interface BalanceModuleCommon<
     callback: SubscriptionCallback<Balances>
   ): Promise<UnsubscribeFn>
 
+  /** Used to provision balances in db while they are fetching for the first time */
+  getPlaceholderBalance(tokenId: TTokenType["id"], address: Address): BalanceJson
+
   /** Fetch balances for this module with optional filtering */
```

Comment:
<div align="center">

![it's beautiful](https://github.com/TalismanSociety/talisman/assets/2741569/73d3139c-9145-430f-b53d-993f2745f5f3)

</div>